babel_runner.py

```diff
@@ -15,9 +15,9 @@
 # Global variables used by pybabel below (paths relative to PROJECT_DIR)
 DOMAIN = "messages"
 COPYRIGHT_HOLDER = "Python Software Foundation"
-LOCALES_DIR = "locales"
-POT_FILE = Path(LOCALES_DIR, f"{DOMAIN}.pot")
 SOURCE_DIR = "python_docs_theme"
```

Comment:
Is the `"python_docs_theme"` on line 13 also an instance of `SOURCE_DIR`? Should it be:
```
INIT_PY = PROJECT_DIR / SOURCE_DIR / "__init__.py"
```